ci-operator/step-registry/ipi/deprovision/deprovision/in-bastion/ipi-deprovision-deprovision-in-bastion-commands.sh

```diff
@@ -47,37 +60,92 @@ function run_scp_from_remote() {
 }
 
 function save_logs() {
-    echo "Copying the Installer logs and metadata to the artifacts directory..."
+    echo "$(date -u --rfc-3339=seconds) - Copying the Installer logs and metadata to the artifacts directory..."
     cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}"
     cp /tmp/installer/metadata.json "${ARTIFACT_DIR}"
 }
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 trap 'save_logs' EXIT TERM
 
+if [[ -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
```

Comment:
I do not think we need this var in deprovision step.

Reply:
Do you mean to always use the installer bin from `which openshift-install`?

Reply:
The same comments as https://github.com/openshift/release/pull/63439#discussion_r2074695491